### ISSUE TITLE
NullPointerException on IntelliJ IDEA after update

### ISSUE THREAD

**Issue by 8armed-monkey** [szpak/gradle-pitest-plugin#60]
Hi,

I am getting the following exception after updating to the new IntelliJ IDEA:

```
Exception in thread "main" java.lang.NullPointerException
        at org.pitest.mutationtest.filter.KotlinFilter$1.apply(KotlinFilter.java:34)
        at org.pitest.mutationtest.filter.KotlinFilter$1.apply(KotlinFilter.java:31)
        at org.pitest.functional.predicate.Not.apply(Not.java:33)
        at org.pitest.functional.predicate.Not.apply(Not.java:23)
        at org.pitest.functional.FCollection.filter(FCollection.java:84)
        at org.pitest.functional.FCollection.filter(FCollection.java:77)
        at org.pitest.mutationtest.filter.KotlinFilter.filter(KotlinFilter.java:27)
        at org.pitest.mutationtest.filter.CompoundMutationFilter.filter(CompoundMutationFilter.java:22)
        at org.pitest.mutationtest.build.MutationSource.createMutations(MutationSource.java:52)
        at org.pitest.mutationtest.build.MutationTestBuilder$2.apply(MutationTestBuilder.java:100)
        at org.pitest.mutationtest.build.MutationTestBuilder$2.apply(MutationTestBuilder.java:97)
        at org.pitest.functional.FCollection.flatMapTo(FCollection.java:56)
        at org.pitest.functional.FCollection.flatMap(FCollection.java:66)
        at org.pitest.mutationtest.build.MutationTestBuilder.createMutationTestUnits(MutationTestBuilder.java:56)
        at org.pitest.mutationtest.tooling.MutationCoverage.buildMutationTests(MutationCoverage.java:277)
        at org.pitest.mutationtest.tooling.MutationCoverage.runReport(MutationCoverage.java:133)
        at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:103)
        at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:45)
        at org.pitest.mutationtest.commandline.MutationCoverageReport.runReport(MutationCoverageReport.java:87)
        at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:45)
```

It works fine before the update. I am currently using this version of IntelliJ IDEA:
```
IntelliJ IDEA 2017.1
Build #IC-171.3780.107, built on March 22, 2017
JRE: 1.8.0_112-release-736-b13 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.11.6
```

I also tried to disable the Kotlin plugin for IntelliJ but it doesn't help.

**Comment by szpak** [comment]
How do you execute PIT from Idea? Which plugin do you?

Is this issue reproducible when running with Gradle (in or outside Idea)?

**Comment by 8armed-monkey** [comment]
Well, actually I ran it from the terminal inside the IntelliJ, so basically I am just using command line. It's the same if I run it with the normal terminal. It's always failing with the same error now (NPE).

I am using pitest Gradle plugin (info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.1.11).

**Comment by 8armed-monkey** [comment]
I am simply running it like this:
`./gradlew clean pitest`

**Comment by szpak** [comment]
The error doesn't seem to be related to gradle-pitest-plugin by PIT itself. Maybe you changed the Kotlin version to 1.1 in the meantime?

Please verify that the issue occurs also in recently released PIT 1.2.0 (just add `pitestVersion` parameter as mentioned in the [README](https://github.com/szpak/gradle-pitest-plugin#plugin-configuration)). If yes, report the issue [upstream](https://github.com/hcoles/pitest/issues) preferably with [SSCCE](http://sscce.org/).
